Makefile

```diff
@@ -222,6 +222,11 @@ lint-pydocstyle: ## lint Python docstrings with pydocstyle
 	@$(COMPOSE_TEST_RUN_APP) pydocstyle
 .PHONY: lint-pydocstyle
 
+lint-mypy: ## lint back-end python sources with mypy
+	@echo 'lint:mypy started…'
+	@$(COMPOSE_TEST_RUN_APP) mypy
+.PHONY: lint-mypy
```

Comment:
Maybe we should include this in `make lint` ?

Reply:
As all warnings are not yet fixed, I voluntarily did not include in the linter to be consistent with CI linter